### PR TITLE
fix failed test caused by PR#777

### PR DESCRIPTION
**Pull request by wangcj05**
PR#777 was created before PR#800, the following test was added by PR#800. PR#777 was tested before PR#800 and it was green, but PR#800 was merged before PR#777, and the following test was not tested with PR#777. 
```
doc/workshop/forwardSampling/Inputs/previous.FS6
```

This PR will also fix the random failure of test occ.xml "framework/ROM/SKLearn.OCC". The reason for the failure is caused by the random shuffling of the data before training the classifier. 

--------
Pull Request Description
--------
##### What issue does this change request address? (Use "#" before the issue to link it, i.e., #42.)
close #824 

##### What are the significant changes in functionality due to this change request?

----------------
For Change Control Board: Change Request Review
----------------
The following review must be completed by an authorized member of the Change Control Board.
- [x] 1. Review all computer code.
- [x] 2. If any changes occur to the input syntax, there must be an accompanying change to the user manual and xsd schema. If the input syntax change deprecates existing input files, a conversion script needs to be added (see Conversion Scripts).
- [x] 3. Make sure the Python code and commenting standards are respected (camelBack, etc.) - See on the [wiki](https://github.com/idaholab/raven/wiki/RAVEN-Code-Standards#python) for details.
- [x] 4. Automated Tests should pass, including run_tests, pylint, manual building and xsd tests. If there are changes to Simulation.py or JobHandler.py the qsub tests must pass.
- [x] 5. If significant functionality is added, there must  be tests added to check this. Tests should cover all possible options.  Multiple short tests are preferred over one large test. If new development on the internal JobHandler parallel system is performed, a cluster test must be added setting, in <RunInfo> XML block, the node ```<internalParallel>``` to True.
- [x] 6. If the change modifies or adds a requirement or a requirement based test case, the Change Control Board's Chair or designee also needs to approve the change.  The requirements and the requirements test shall be in sync.
- [x] 7. The merge request must reference an issue.  If the issue is closed, the issue close checklist shall be done.
- [x] 8. If an analytic test is changed/added is the the analytic documentation updated/added?

### CODE REVIEW

**Comment by wangcj05** [comment]
@alfoa Can you take a look at this PR. Thanks a lot.

File: tests/framework/ROM/SKLearn/occ.xml [hunk 1]
```diff
@@ -45,7 +45,9 @@
         <SKLtype>svm|SVC</SKLtype>
         <fit_intercept>True</fit_intercept>
         <normalize>True</normalize>
+        <random_state>0</random_state>
       </estimator>
+      <random_state>0</random_state>
     </ROM>
```
Comment (alfoa) [inline tests/framework/ROM/SKLearn/occ.xml]: Do we need to set it in both places? 
Shouldn't be enough to set it in the multiclass and send it to all the estimators?

Comment (wangcj05) [reply tests/framework/ROM/SKLearn/occ.xml]: I think I agree with you, one is good to all the estimators.

Comment (wangcj05) [reply tests/framework/ROM/SKLearn/occ.xml]: I do test it, removing the inside random_state does not break the test. I will remove it.

**Comment by mandd** [comment]
do we still have a problem if we set:
`     <PostProcessor name="Stat" subType="BasicStatistics">
      <expectedValue prefix="mean">X1,X2,Y1,Y2</expectedValue>
      <sensitivity prefix="sen">
        <targets>X1,X2,Y1,Y2</targets>
        <features>X1,X2,Y1,Y2</features>
      </sensitivity>
    </PostProcessor>`

**Comment by wangcj05** [comment]
> do we still have a problem if we set:
> 
> ```
>    <PostProcessor name="Stat" subType="BasicStatistics">
>       <expectedValue prefix="mean">X1,X2,Y1,Y2</expectedValue>
>       <sensitivity prefix="sen">
>         <targets>X1,X2,Y1,Y2</targets>
>         <features>X1,X2,Y1,Y2</features>
>       </sensitivity>
>     </PostProcessor>`
> ```

@mandd Yes, we do have a problem with the results. When we include all the variables in the targets and features, there will be a collinearity problem which will ruin the sensitivity calculations.

**Comment by wangcj05** [comment]
@alfoa It seems we have another problems with Test Ubuntu 16. This PR only contains tests changes, and should not affect other tests since there is no code changes.

**Comment by wangcj05** [comment]
@alfoa Can you take a look at the failed test on Ubuntu 16? Is it caused by the memory issue?

**Comment by alfoa** [comment]
PR approved...

**Comment by wangcj05** [comment]
Thanks.

________________________________

**Pull request merged**